docs/developer-docs/integrations/sns/launching/launch-steps-1proposal.md

```diff
@@ -0,0 +1,99 @@
+# SNS launch steps
+
+## Overview
+At a high level, the stages for launching an SNS in production are explained [here](../launching/launch-summary.md).
+
+This article lists the technical commands and steps a developer needs to complete the stages for an SNS launch.
+
+At a low level, the [SNS local testing repository](../testing/testing-locally.md) guides you
+through the same, with the difference that the commands here target the canisters on the mainnet.
+
+## Requirements
+
+### 1. IC SDK
+
+See: [installing the IC SDK](../../../setup/install).
+
+### 2. `ic-admin`
+
+See: [installing the `ic-admin`](../../../setup/ic-admin.md).
+
+### 3. `sns` CLI
+
+:::note
+The version of the sns CLI that is bundled with your dfx version may not have the latest commands described in the Usage section. If needed, it is recommended to build and use the sns CLI tool yourself.
+:::
+
+```bash
+git clone git@github.com:dfinity/ic.git
+cd ic
+bazel build //rs/sns/cli:sns
+ls bazel-bin/rs/sns/cli/sns 
+```
+## Stages
+
+### 1. Dapp developers choose the initial parameters of the SNS for a dapp
+
+Typically, dapp developers choose initial parameters that will be used in subsequent proposals.
+
+### 2. Dapp developers submit NNS proposal to create SNS
+
+Anyone who owns an eligible NNS neuron with enough stake can submit an NNS proposal to create an
+SNS for a given dapp.
+Of course it is crucial to set the right parameters in this proposal.
+You can also find an example how this command is used in the SNS local testing
+[here](https://github.com/dfinity/sns-testing/blob/main/open_sns_sale.sh#L11-L26).
+
+To create such a proposal, a common path is to use `ic-admin` and run the following:
+[TODO: update command]
```

Comment:
```suggestion
To create such a proposal, a common path is to use `sns-cli` and run the following:
```
dfx sns propose --network ic --neuron $NEURON_ID sns_init.yaml
```
```

Reply:
Will be added in next commit. Would be great if you can check that this makes sense.